content/docs/concepts/flops/overview.md

```diff
@@ -0,0 +1,158 @@
+The metrics they share with the root are distilled and no longer contain sensitive individual metadata.
+This is an ideal environment for FL because this layout supports privacy on a structural level.
+Oakestra's slim open-source code base allows for easy extension, further development, and experimentation with FL features.
+
+## FLOps
+FLOps is an [Oakestra Addon](TODO) that enables streamlined and accelerated practical FL workflows via:
+
+{{< details "Automated FL Augmentation/Containerization" >}}
+  FLOps allows users to perform FL by simply providing a link to their ML Git repository.
+  This repository code needs to satisfy some simple structural prerequisites.
+  It gets automatically augmented by FLOps to support FL.
+  FLOps creates a containerized image with all necessary dependencies to do FL training.
+  These images are automatically built and adhere to best practices, ensuring they are as fast and lightweight as possible.
+  FLOps can build these images for multiple different target platforms.
+  Thus, FL components can run on ARM edge devices like Raspberry Pis or Nvidia Jetsons.
+  FLOps enables FL on all devices that support containerization technologies like Docker or containerd.
+  This approach eliminates the need for tedious device setup and the struggle to configure heterogeneous dependencies to match the training requirements. 
+{{< /details >}}
+
+{{< details "Automated Orchestration of FL Components and Auxiliaries" >}}
+  FLOps automatically performs FL training based on the user-requested configuration, such as:
+  - Resource Requirements
+  - Number of Training Rounds
+  - The FL Algorithm/Method
+  - Minimum Number of Learners
+{{< /details >}}
+
+{{< details "Elevated Observability & Tracking via MLOps" >}}
+  During runtime, users can observe this training process via a sophisticated GUI, which allows users to monitor, compare, store, export, share, and organize training runs, metrics, and trained models.
+{{< /details >}}
+
+{{< details "Accessible Inference Serving" >}}
+  FLOps can automatically build inference servers based on the trained model.
+  This inference server can be pulled as a regular image.
+  FLOps can also directly deploy this trained-model image as an inference server on Oakestra worker nodes.
+{{< /details >}}
+
+{{< details "Combine latest Frameworks & Tools" >}}
+  FLOps combines state-of-the-art solutions from different fields in novel ways instead of reinventing custom subpar solutions from scratch.
+  To the best of our knowledge, FLOps is the first work that combines Flower with MLflow, allows hierarchical FL, and automatically converts ML code into FL-enabled containerized images.
+
+  Every third-party component and dependency has been carefully analyzed and compared with possible alternatives.
+  The following is a selection of tools and frameworks that power FLOps.
+
+  ### [Flower](https://flower.ai/) 
```

Comment:
Dont add reference in title as it makes internal section referencing harder. Instead refer in first occurence within text